src/Apps/Sell/Routes/__tests__/FrameRoute.jest.tsx

```diff
@@ -35,10 +51,10 @@ beforeEach(() => {
       push: mockPush,
       replace: mockReplace,
     },
-    match: { location: { pathname: "/sell/submissions/submission-id/frame" } },
+    match: { location: { pathname: "/sell/submissions/externalId/frame" } },
```

Comment:
Is externalId here the same as submission-id? Let's use submission-id and stay consistent